### DISCUSSION TITLE
`group` utility doesn't work with <i> tag

### DISCUSSION THREAD

**Discussion by Harizriz** [tailwindlabs/tailwindcss#11101] [Help] [answered]
Here is a code example. I'm using fontawesome6 for the icon.

```
<button
  class="group h-10 w-10 rounded-full border-[1px] border-white hover:border-cyan-gradient"
>
  <i
    class="fa-brands fa-facebook-f group-hover:fill-cyan-gradient"
    style="color:#ffffff"
  />
</button>
```

Problem:
I want to group the button and the icon so that on `:group-hover` both will change to a cyan color. However, the group-hover psuedo class in the `<i>` tag doesn't work correctly. It won't change it's color when I hover over the button. It will only change its color when it's hovered over it.

**Comment by wongjn** [comment, marked as answer]
Please consider providing an example that reproduces your issue, like a [Tailwind Play](https://play.tailwindcss.com).

From what you've provided so far, it seems like the icon should stay white, even when hovering over it. You would need to change the `group-hover:fill-` to `group-hover:!text-`, so that the `color` value in the `style` attribute is overridden: https://play.tailwindcss.com/Y1wIfADnIF
